multimedia/views.py

```diff
@@ -3,5 +3,10 @@
 
 
 def book_list(request):
-    books = Book.all()
-    return render(request, 'catalog/list.html', {'data': books})
+    books = Book.objects.all()
```

Comment:
I have overwritten the `BookManager.all` method in  https://github.com/exclusive-gix/cs2102-store/blob/feature/book/multimedia/models.py#L33-L47